core/block_rendering_rewrite/block_render_info.js

```diff
@@ -127,6 +127,8 @@ Blockly.BlockRendering.RenderInfo.prototype.measure_ = function() {
  */
 Blockly.BlockRendering.RenderInfo.prototype.createRows_ = function() {
   var activeRow = new Blockly.BlockRendering.Row();
+  this.topRow = this.createTopRow_();
```

Comment:
Move these above the `activeRow` line, so that it's clear that the `activeRow` will come after these.

Do we still need `this.topRow` now that it's not as distinct?

Reply:
We use it in the draw function and draw highlight function so I think it is probably worth it to keep it around? It seems cleaner than getting the first element in the list of rows every time.